packages/react-component-library/cypress/specs/NumberInputE/index.spec.ts

```diff
@@ -37,6 +37,12 @@ describe('NumberInputE', () => {
     ['100.123.456', '100.123456'],
     ['100.123', '100.123'],
     ['100.invalid456', '100.456'],
+    ['100.25{home}.25', '25100.25'],
+    ['-100.123', '-100.123'],
+    ['-100.1-23', '-100.123'],
+    ['100.1-23', '100.123'],
+    ['12.34{selectall}56.78', '56.78'],
+    ['-12.34{selectall}-56.78', '-56.78'],
   ]
 
   TYPED_VALUE_CASES.forEach(([textToType, expectedValue]) => {
```

Comment:
Unfortunately, I haven't found a way to test pasting in Cypress (typing Ctrl-V doesn't work, and you can simulate the event but that is only the event and not actually pasting).

Reply:
Is [this](https://egghead.io/blog/handling-copy-and-paste-in-cypress) close enough?

```javascript
cy.get("[type='email']").invoke('val', textToPaste)
```

Reply:
No, that just manipulates that value attribute using jQuery unfortunately :disappointed: